ShopProjectHomeWork/ShopProjectHomeWork/SearchScreen/SearchScreenScrollContentView.swift

```diff
@@ -0,0 +1,100 @@
+//
+//  ScrollViewContentView.swift
+//  ShopProjectHomeWork
+//
+//  Created by Григоренко Александр Игоревич on 09.10.2022.
+//
+
+import UIKit
+
+/// Экран поиска товара.
+final class SearchScreenScrollContentView: UIView {
+
+    // MARK: - private visual components
+
+    private var macbookView = GoodsView()
+    private var clockView = GoodsView()
+    private var brownCaseView = GoodsView()
+
+    // MARK: - private properties
+
+    private var tapActionHandler: ((_ handler: [UIImage?]) -> Void)?
+
+    // MARK: - Инициализатор
+
+    init(frame: CGRect = CGRect(),
+         action: ((_ handler: [UIImage?]) -> Void)? = nil
+    ) {
+        self.tapActionHandler = action
+        super.init(frame: frame)
+        configureView()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    // MARK: - private funcs
+
+    private func configureView() {
+        addMacbookCaseView()
+        addClockBeltView()
+        addBrownCaseView()
+    }
+
+    private func addMacbookCaseView() {
+        macbookView = GoodsView(
+            frame: CGRect(
+                x: frame.origin.x,
+                y: 0,
+                width: 150,
+                height: 200
+            ),
+            images: [UIImage(named: "case1"),
+                             UIImage(named: "case2"),
+                             UIImage(named: "case3")],
```

Comment:
```suggestion
                             UIImage(named: "")],
```
все текстовки в константы